google-cloud-bigtable/src/main/java/com/google/cloud/bigtable/data/v2/stub/metrics/CompositeTracer.java

```diff
@@ -124,9 +123,10 @@ public void attemptCancelled() {
     }
   }
 
-  public void attemptFailed(Throwable error, Duration delay) {
```

Comment:
Instead of deleting this method, should attemptFailled call attemptFailedDuration?

Reply:
We moved all references in gax to `attemptFailedDuration`
https://github.com/googleapis/sdk-platform-java/pull/1872

However it still makes sense to keep both. I'll push a commit with this.

Reply:
I noticed our tracking sheet has a new comment in there. Confirmed it should be as you pointed out.